### PR TITLE
[GSK-1476] introducing debug_description in test decorator

### PR DESCRIPTION
**Pull request by rabah-khalek**
The idea is to describe what the debug does for every test when a user clicks on the button

### CODE REVIEW

**Comment by rabah-khalek** [comment]
@Googleton, I introduced the default message, What's left is to propagate it for every test with a custom message.
Could you please test if you can retrieve it on the UI without any issues? I created the DTOs but just want to be sure I haven't forgotten anything. Thanks!

**Comment by Inokinoki** [comment]
Preview:

image

File: frontend/src/views/main/project/SuiteTestExecutionCard.vue [hunk 1]
```diff
@@ -80,6 +80,40 @@
...
+      <v-dialog
+          v-model="debugDescDialog"
+          width="auto"
+      >
+        <v-card>
+          <v-card-title>
+            Debug the {{ suiteTest.test?.displayName }} test
+          </v-card-title>
+          <v-card-text>
+            <!-- //NOSONAR --><p v-html="suiteTest.test.debugDescription"/>
```
Comment (Googleton) [inline frontend/src/views/main/project/SuiteTestExecutionCard.vue:93]: Why the comment ?

**Review by Inokinoki — APPROVED** [review]
Nice work @rabah-khalek for the debug description messages!
Could you plz test to start a new instance without existing DB from `main` and connect an internal ML worker (so that the test function does not contain debug description), and then restart both DB and worker from this branch, to see whether the debug descriptions are updated or not.
I tested locally, it seems to be working now.

**Review by Googleton — APPROVED** [review]

**Comment by rabah-khalek** [comment]
@Inokinoki @Googleton the backend build seems to be failing, could you please have a look? I hope it's an easy fix.

**Comment by Inokinoki** [comment]
> @Inokinoki @Googleton the backend build seems to be failing, could you please have a look? I hope it's an easy fix.

Checked, it seems to be random network issue in the last CI. This one passes

**Pull request merged**